exercises/4_6/index.html

```diff
@@ -0,0 +1,138 @@
+
+			header1.appendChild(headerTitle);
+			document.querySelector("body").appendChild(header1);
+
+			// Adicione a tag div com a classe main-content como filho da tag body;
+			let div1 = document.createElement("div");
+			div1.className = "main-content";
+			document.querySelector("body").appendChild(div1);
+
+			// Adicione a tag div com a classe center-content como filho da tag div criada no passo 2;
+			let div2 = document.createElement("div");
+			div2.className = "center-content";
+			document.querySelector("div").appendChild(div2);
+
+			// Adicione a tag p como filho do div criado no passo 3 e coloque algum texto;
+
+			let text = document.createElement("p");
+			text.innerHTML = "FILHO DA DIV!!!";
+			document.querySelector(".center-content").appendChild(text);
+
+			// Adicione a tag div com a classe left-content como filho da tag div criada no passo 2;
+
+			let leftContent = document.createElement("div");
+			leftContent.className = "left-content";
+			document.querySelector(".main-content").appendChild(leftContent);
+
+			// Adicione a tag div com a classe right-content como filho da tag div criada no passo 2;
+
+			let rightContent = document.createElement("div");
+			rightContent.className = "right-content";
+			document.querySelector(".main-content").appendChild(rightContent);
+
+			// Adicione uma imagem com src configurado para o valor https://picsum.photos/200 e classe small-image. Esse elemento deve ser filho do div criado no passo 5;
+
+			let image = document.createElement("img");
+			image.src = "https://picsum.photos/200";
+			image.className = "small-image";
+			document.querySelector(".left-content").appendChild(image);
+
+			// Adicione uma lista não ordenada com os valores de 1 a 10 por extenso, ou seja, um, dois, três, … como valores da lista. Essa lista deve ser filha do div criado no passo 6;
+			
+			let list = document.createElement("ul");
+			list.className = "unordered";
+			document.querySelector(".right-content").appendChild(list);
+
+			let stringArray = ["um", "dois", "três", "quatro", "cinco", "seis", "sete", "oito", "nove", "dez"];
+				for (index in stringArray) {
+					let listItem = document.createElement("li");
+					listItem.innerHTML += stringArray[index];
+					document.querySelector(".unordered").appendChild(listItem);
+				}
+			
+			// Adicione 3 tags h3, todas sendo filhas do div criado no passo 2.
+
+			for (header3 = 0; header3 < 3; header3 += 1) {
+				let header3 = document.createElement("h3");
+				header3.className = "description";
+				document.querySelector(".main-content").appendChild(header3);
+			}
+
+			// Parte 2 
+			header1.className = "title";
+			
+			document.querySelector(".main-content").removeChild(leftContent);
+
+			document.querySelector(".right-content").style.margin = "auto";
+
+			document.querySelector(".center-content").parentElement.style.backgroundColor = "green";
+
+			for (item = 0; item < 2; item += 1) {
```

Comment:
Mandou bem nesse `for`, não tinha pensado desse jeito!

Reply:
Valeu! #gambiarra